lib/commands/project/project.js

```diff
@@ -383,6 +383,24 @@ function testGenerate(directory, options, cb) {
   var testModule = options.testModule || TESTMODULES[0];
   var assertionFormat = options.assertionFormat || ASSERTIONTYPES[0];
 
+  findProjectFile(directory, options, function(err, projPath) {
+    var projectFile = fs.readFileSync(projPath, 'utf8');
+    var projectJson = JSON.parse(projectFile);
+
+    projectJson.dependencies['z-schema'] = '^3.12.0';
+    projectFile = JSON.stringify(projectJson);
+
+    fs.writeFileSync(projPath, projectFile);
+
+    spawn('npm', ['install'], process.cwd(), function(err) {
```

Comment:
the npm install task is [repeated code](https://github.com/noahdietz/swagger-node/blob/test-generation/lib/commands/project/project.js#L98-L103). Try encapsulating it in one function and calling it twice instead of repeating it